lang/cs/Org.Apache.REEF.DotNet.sln

```diff
@@ -109,4 +109,7 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {36166814-E0E6-450B-A405-25925D9A48E2}

```

Comment:
What's this new item in the `.sln` for?

Reply:
I am not sure how that was added - it likely an artifact of what I did previously. I have removed those lines for now.